spec/controllers/activitypub/outboxes_controller_spec.rb

```diff
@@ -7,9 +7,23 @@
     Fabricate(:status, account: account)
   end
 
-  describe 'GET #show' do
+  describe 'GET #show for routing version v1' do
     before do
-      get :show, params: { account_username: account.username }
+      get :show, params: { account_type: 'v1_trainer', account_username: account.username }
+    end
+
+    it 'returns http success' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'returns application/activity+json' do
+      expect(response.content_type).to eq 'application/activity+json'
+    end
+  end
+
+  describe 'GET #show for routing version v2' do
+    before do
+      get :show, params: { account_type: 'trainer', account_no: account.trainer_no }
```

Comment:
this *should* fail as-is probably because `account` is a v1 trainer so the v2 routing shouldn't work

you'll need to `Fabricate(:trainer)` in addition to the `Fabricate(:account)` to test for v2 trainers

Reply:
oh that probably explains why it does fail then.